Makefile

```diff
@@ -5,36 +5,31 @@ all:
 	@echo make restart
 	@echo make keyboard-dev
 
-install-audio-receiver:
-	wget -q https://github.com/nicokaiser/rpi-audio-receiver/archive/main.zip
-	unzip main.zip
-	rm main.zip
-	
-	cd rpi-audio-receiver-main;	sudo ./install.sh
-
-dependencies: install-audio-receiver
+dependencies:
```

Comment:
`rpi-audio-receiver` installation doesn't work as smoothly as it used to, I think because of updates to `shairport` (the AirPlay module). Because of that, and because duplication was bad, I'm removing this installation from here and just leaving it in the README for more manual use.